group_vars/all/vars.sample

```diff
@@ -45,6 +45,7 @@ windows_update: true
 # Windows server timezone
 # Important that this value matches the timezone used by UCP servers for certificate validation
 windows_tz: 'Pacific Standard Time'
+windows_ntp_servers: "10.60.1.123 0.us.pool.net.org 1.us.pool.net.org"
 windows_docker_drive: 'D'
```

Comment:
we already have a variable to name the ddocker data drive and this is disk2.  It should be configured in one of the existing group variables files (probably wndows_box.yml)